app/src/main/java/org/jellyfin/androidtv/util/profile/ExoPlayerProfile.kt

```diff
@@ -193,4 +180,32 @@ class ExoPlayerProfile(
 			subtitleProfile("idx", SubtitleDeliveryMethod.Embed)
 		)
 	}
+
+	companion object {
+		fun getDownmixSupportedAudioCodecs(): Array<String> =
+			arrayOf(
+				CodecTypes.AAC,
+				CodecTypes.MP3,
+				CodecTypes.MP2
+			)
+
+		/**
+		 * Returns all audio codecs used commonly in video containers.
+		 * This does not include containers / codecs found in audio files
+		 */
+		fun getAllSupportedAudioCodecs(): Array<String> =
```

Comment:
Make private and move to a value instead (`val downmixSupportedAudioCodecs = arrayOf(...)`)

Reply:
also move both to before the `init{}` block